docs/whats-new/2020-09.md

```diff
@@ -0,0 +1,121 @@
+- [Basic Queries (LINQ to XML)](../standard/linq/basic-queries-linq-to-xml.md) - Add orphans to TOC
+
+## .NET Core
+
+### New articles
+
+- [Options pattern in .NET](../core/extensions/options.md) - Port over .NET Options Pattern article to finalize Microsoft Extensions fundamentals
+- [Implement a custom logging provider in .NET](../core/extensions/custom-logging-provider.md) - Port logging articles - adapt to .NET docs
+- [High-performance logging in .NET](../core/extensions/high-performance-logging.md) - Port logging articles - adapt to .NET docs
+- [Logging providers in .NET](../core/extensions/logging-providers.md) - Port logging articles - adapt to .NET docs
+- [Logging in .NET](../core/extensions/logging.md) - Port logging articles - adapt to .NET docs
+- [Dependency injection guidelines](../core/extensions/dependency-injection-guidelines.md) - Introduce and port over dependency injection articles for Microsoft Extensions
+- [Tutorial: Use dependency injection in .NET](../core/extensions/dependency-injection-usage.md) - Introduce and port over dependency injection articles for Microsoft Extensions
+- [Dependency injection in .NET](../core/extensions/dependency-injection.md) - Introduce and port over dependency injection articles for Microsoft Extensions
+- [.NET Generic Host](../core/extensions/generic-host.md) - Add Generic Host article
+- [Configuration providers in .NET](../core/extensions/configuration-providers.md) - Config fundamentals
+- [Configuration in .NET](../core/extensions/configuration.md) - Config fundamentals
+- [Implement a custom configuration provider in .NET](../core/extensions/custom-configuration-provider.md) - Config fundamentals
+- [Collect diagnostics in containers](../core/diagnostics/diagnostics-in-containers.md) - Add documentation on diagnosing .NET Core performance issues in containers
+- [Breaking changes in Windows Presentation Framework (WPF)](../core/compatibility/wpf.md) - WinForms and WPF SDK - breaking change
+- [Debug Linux dumps](../core/diagnostics/debug-linux-dumps.md) - Linux dump debugging docs
+- [SOS installer (dotnet-sos)](../core/diagnostics/dotnet-sos.md) - Linux dump debugging docs
+- [Symbol downloader (dotnet-symbol)](../core/diagnostics/dotnet-symbol.md) - Linux dump debugging docs
+- [Single file deployment and executable](../core/deploying/single-file.md) - Conceptual document for single file publish option
+- [The evolution of .NET Core to .NET 5](../core/dotnet-five.md) - Adding What is .NET 5 article
+
+## .NET for Apache Spark
+
+### Updated articles
+
+- [Tutorial: Get started with .NET for Apache Spark](../spark/tutorials/get-started.md) - Spark | Update getting started tutorial
+
+## .NET fundamentals
+
+### New articles
+
+- [Configuration files for code analysis rules](../fundamentals/code-analysis/configuration-files.md) - Add documentation for Global AnalyzerConfig
```

Comment:
I'm not sure why the code quality rules work doesn't show up here.

https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/
https://github.com/dotnet/docs/pull/20702

Reply:
There's a filter that ignores PRs with more than 250 files. Those have been PRs that make automated script changes.

Is there a landing page to send readers to? I'll add that one by hand.

Reply:
Yes, I think this is a good page:

https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/

Reply:
added it.